unlock-app/src/components/content/EventContent.tsx

```diff
@@ -0,0 +1,201 @@
+  }
+
+  return (
+    <AppLayout showLinks={false} authRequired={false} title="">
+      <Head>
+        <title>{pageTitle('Event')}</title>
+      </Head>
+      <div className="md:w-3/4 m-auto">
+        {(!router.query.lockAddress || !router.query.network) && (
+          <p>
+            Use Unlock to sell tickets for your event:{' '}
+            <Link className="underline" href="/locks/create">
+              start by deploying your contract.
+            </Link>
+          </p>
+        )}
+        {router.query.lockAddress && router.query.network && (
+          <EventDetails
+            lockAddress={router.query.lockAddress.toString()}
+            network={parseInt(router.query.network.toString(), 10)}
+          />
+        )}
+      </div>
+    </AppLayout>
+  )
+}
+
+interface EventDetailsProps {
+  lockAddress: string
+  network: number
+}
+
+export const EventDetails = ({ lockAddress, network }: EventDetailsProps) => {
+  const { account } = useAuth()
+
+  const { data: metadata, isInitialLoading: isMetadataLoading } = useMetadata({
+    lockAddress,
+    network,
+  })
+
+  if (isMetadataLoading) {
+    return <LoadingIcon></LoadingIcon>
+  }
+
+  if (!metadata?.attributes) {
+    return <p>Not an event!</p>
+  }
+
+  const eventData = formatEventData(metadata)
+
+  const addToCalendar = () => {
+    alert('AA')
+  }
+
+  const shareOnTwitter = () => {
+    alert('TT')
+  }
+
+  return (
+    <main className="grid md:grid-cols-[minmax(0,_1fr)_300px] gap-8 mt-8">
+      <section className="">
+        <h1 className="text-5xl md:text-7xl font-bold mb-4">
+          {eventData.title}
+        </h1>
+        <p className="flex flex-rows gap-2 mb-4">
+          <span className="text-brand-gray">Ticket contract</span>
+          <AddressLink
+            lockAddress={lockAddress}
+            network={network}
+          ></AddressLink>
+        </p>
+        <ul
+          className="bold text-xl md:text-2xl mb-6"
+          style={{ color: eventData.accentColor }}
+        >
+          <li className="mb-2">
+            <FaCalendar className="inline mr-2" />
+            {eventData.date}
+          </li>
+          <li className="mb-2">
+            <FaClock className="inline mr-2" />
+            {eventData.time}
+          </li>
+          <li className="mb-2">
+            <Link
+              target="_blank"
+              href={`https://www.google.com/maps/search/?api=1&query=${eventData.address}`}
+            >
+              <GoLocation className="inline mr-2" />
+              {eventData.address}
+            </Link>
+          </li>
+        </ul>
+        <div className="markdown">
+          {/* eslint-disable-next-line react/no-children-prop */}
+          <ReactMarkdown children={eventData.description}></ReactMarkdown>
+        </div>
+      </section>
+      <section className="flex flex flex-col items-center">
+        <img alt={eventData.title} className="mb-5" src={eventData.image} />
```

Comment:
Add aspect ratio, overflow, cover, and max size classes to avoid any overflow issues.

Reply:
The problem is that this is the NFT image and we don't actually have these details :/

Reply:
We don't require them. I'm only asking to add some defensive styling to avoid any responsiveness issues, not actually setting any size/width/height explicitly.

Reply:
HA! Ok let me add that!